packages/user_packages/iotcc_mqtt.py

```diff
@@ -97,6 +100,9 @@ def run(self, registry):
                     self.iotcc.set_properties(self.iotcc_edge_system, {"key1": "value1", "key2": "value2"})
                     break
                 except Exception:
```

Comment:
Should we check for various exceptions? For example, if it is a version mismatch, it might not make sense to retry?

Reply:
Agreed, Check_version is recently introduced. I`m not sure how to handle it in user_code. Can you mark a separate issue for this change?

Reply:
Agreed. Since this is in user example code, we can handle it out of this PR and even after code freeze date.